pydis_site/apps/api/viewsets/bot/off_topic_channel_name.py

```diff
@@ -21,6 +21,10 @@ class OffTopicChannelNameViewSet(DestroyModelMixin, ViewSet):
     If the `random_items` query parameter is given, for example using...
         $ curl api.pythondiscord.local:8000/bot/off-topic-channel-names?random_items=5
     ... then the API will return `5` random items from the database.
+    If the `mark_used` query parameter is given like...
+        $ curl api.pydis.local:8000/bot/off-topic-channel-names?random_items=5&mark_used=true
+    ... then the API will mark returned `5` items `used`.
+    When running out of names, API will mark all names to not used and start new round.
```

Comment:
Since you're removing this part, just move some of this information about starting new rounds into the docstring for the `list` function itself.